roc/roc_test.go

```diff
@@ -0,0 +1,129 @@
+package roc
+
+import "testing"
+
+func fail(expected interface{}, got interface{}, t *testing.T) {
+	t.Errorf("Mismatch, expected: %v, got: %v", expected, got)
+	t.Fail()
+}
+
+func Test_roc_address_init(t *testing.T) {
```

Comment:
I suggest to split roc_test.go into multiple files as well.

Reply:
done